pages/common/git-sizer.md

```diff
@@ -0,0 +1,16 @@
+# git sizer
+
+> Computes various Git repository size metrics and alerts you to any that might cause problems or inconvenience.
+> Change to the directory containing your repository before running this command.
+
+- Report only statistics that have >0 level of concern:
```

Comment:
Let's change this to `Report only statistics that have a level of concern greater than 0`